packages/core/src/render3/di.ts

```diff
@@ -168,7 +168,8 @@ function bloomHashBit(type: viewEngine.Type<any>): number|null {
 export function bloomFindPossibleInjector(
     startInjector: LNodeInjector, bloomBit: number): LNodeInjector|null {
   // Create a mask that targets the specific bit associated with the directive we're looking for.
-  // This will be a number between 0 and 31, corresponding to a bit position in a 32 bit integer.
+  // JS bit operations are 32 bits, so this will be a number between 2^0 and 2^31, corresponding
```

Comment:
nit: not sure if `a number between 2^0 and 2^31` is the most helpful `equal to 2^n with n ranging from 0 to 31` is more correct here

Reply:
Not sure I agree here. I think adding an `n` obfuscates it a bit for me.

Reply:
I also find it easier with `a number between 2^0 and 2^31`